src/services/index.ts

```diff
@@ -16,4 +20,30 @@ export const confirmUserEmail = async (payload: any) => {
   return userData;
 };
 
+export const updateMarksAndModels = async (
+  marks: ITransformedMarks[],
+  models: any,
+  bodyTypes: string[]
+) => {
+  await updateBodyTypes(bodyTypes);
+  for (const mark of marks) {
+    const markMaket = { name: mark.name };
+    const savedMark: any = await updateMarks(markMaket);
+    const markId = savedMark.id;
+    if (mark.name === 'BMW' || mark.name === 'Mercedes') {
```

Comment:
If you have to hardcode something, please write comments to describe why.

Reply:
fixed, I left comment here